org.natron.Natron.json

```diff
@@ -0,0 +1,856 @@
+{
+    "id": "org.natron.Natron",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "Natron",
+    "rename-desktop-file": "natron.desktop",
```

Comment:
Could upstream name these correctly?

Reply:
you mean lowercase the executable name?

Reply:
No, using reverse DNS for desktop file. It's required by [the specification](https://standards.freedesktop.org/desktop-entry-spec/latest/ar01s02.html#desktop-file-id), just nothing enforces it.